core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -124,7 +124,7 @@ private SqlLibraryOperators() {
    *
    * <p>It has similar semantics to standard SQL's
    * {@link SqlStdOperatorTable#SUBSTRING} function but different syntax. */
-  @LibraryOperator(libraries = {ORACLE})
+  @LibraryOperator(libraries = {ORACLE, BIGQUERY})
```

Comment:
Add Hive And Spark as well

Reply:
Once bigquery is handled with all variations, I will add spark and hive in as well.